Jenkinsfile

```diff
@@ -139,8 +139,14 @@ pipeline {
             steps {
                 container('dind') {
                     sh '''
+                        apk add jq
                         docker buildx create --name="container" --driver=docker-container --bootstrap
                         echo ${DOCKERHUB_PRESTODB_CREDS_PSW} | docker login --username ${DOCKERHUB_PRESTODB_CREDS_USR} --password-stdin
+                        IMAGE_SHAS=$(docker buildx imagetools inspect --raw "${DOCKER_IMAGE}" | jq -r '.manifests.[].digest')
+                        ORG_IMG_NAME=${DOCKER_IMAGE%:*}
```

Comment:
does this work only for bash?
if so, please add #!/bin/bash -ex, like in like 142
`sh '''#!/bin/bash -ex`

Reply:
Nope, I verified it in a `docker:24.0.7-dind-alpine3.19` container, which is using busybox sh.

Reply:
so this works in `sh`?
` ORG_IMG_NAME=${DOCKER_IMAGE%:*}`

Reply:
we are using `docker:24.0.7-dind-alpine3.19` image to run those commands, right? If that's the case, I verified that the command works in that image.
```sh
/ # echo $DOCKER_IMAGE
public.ecr.aws/oss-presto/presto:0.286-edge13-20240106192158-2ddbebb
/ # ORG_IMG_NAME=${DOCKER_IMAGE%:*}
/ # echo ${ORG_IMG_NAME}
public.ecr.aws/oss-presto/presto
```

Reply:
BTW, it's using `ash`:
```
docker run -ti --rm docker:24.0.7-dind-alpine3.19 ash
/ # DOCKER_IMAGE=public.ecr.aws/oss-presto/presto:0.286-edge13-20240106192158-2ddbebb
/ #  ORG_IMG_NAME=${DOCKER_IMAGE%:*}
/ #  echo ${ORG_IMG_NAME}
public.ecr.aws/oss-presto/presto
```